Makefile.am

```diff
@@ -1,302 +0,0 @@
-# Copyright 2013-present Barefoot Networks, Inc.
```

Comment:
git likely does not remove your local files once you've update the branch. You can run `git clean -dxf` after you rebased your master.

Removing them from .gitignore is a good catch!